Gemfile.lock

```diff
@@ -122,7 +125,10 @@ GEM
     nio4r (2.5.9)
     nokogiri (1.15.4-x86_64-darwin)
       racc (~> 1.4)
-    pg (1.5.3)
```

Comment:
@GowthamK01 Should we remove `pg` ?

Reply:
@gomathi-mallow Have checked and reverted the change.